src/runtime/rpc/rpc_module.cc

```diff
@@ -124,7 +124,8 @@ class RPCModuleNode final : public ModuleNode {
   PackedFunc GetTimeEvaluator(const std::string& name,
                               TVMContext ctx,
                               int number,
-                              int repeat) {
+                              int repeat,
+                              int min_repeat_ms) {
```

Comment:
Does this break some current tests if we do not give a default value for `min_repeat_ms`?

Reply:
I added a default argument to python side